### PR TITLE
Fix spelling mistakes in code found by codespell

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
@edsantiago @TomSweeneyRedHat PTAL

**Comment by rhatdan** [comment]
Prompted by https://github.com/containers/libpod/pull/5404

File: libpod/config/config.go [hunk 1]
```diff
@@ -437,7 +437,7 @@ func probeConmon(conmonBinary string) error {
 // with cgroupsv2. Other OCI runtimes are not yet supporting cgroupsv2. This
 // might change in the future.
 func NewConfig(userConfigPath string) (*Config, error) {
-	// Start with the default config and interatively merge fields in the system
+	// Start with the default config and interactively merge fields in the system
```
Comment (vrothberg) [inline libpod/config/config.go]:
```suggestion
	// Start with the default config and iteratively merge fields in the system
```

Comment (edsantiago) [reply libpod/config/config.go]: I feel so embarrassed not to have caught that. Thank you.

Comment (rhatdan) [reply libpod/config/config.go]: Fixed

**Pull request merged**